resource/subfunctions/appc_session.md

```diff
@@ -0,0 +1,259 @@
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AlterOperationError`<br>
+  Raises `AlterOperationError` when the **z/OS userid** supplied cannot be altered because it does **NOT** exist.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** provided contains one or more **unknown** traits.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.alter_appc_session("TSTNET", "TSTLOCLU", "TSTPRTLU", traits={"session:locked": True})
+{'securityResult': {'resource': {'name': 'TSTNET.TSTLOCLU.TSTPRTLU', 'class': 'APPCLU', 'operation': 'set', 'requestId': 'ResourceRequest', 'info': ['Definition exists. Add command skipped due  to precheck option'], 'commands': [{'safReturnCode': 0, 'returnCode': 0, 'reasonCode': 0, 'image': 'RALTER  APPCLU             (TSTNET.TSTLOCLU.TSTPRTLU)  SESSION  (   LOCK        )'}]}, 'returnCode': 0, 'reasonCode': 0}}
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"TSTNET.TSTLOCLU.TSTPRTLU",
+      "class":"APPCLU",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "info":[
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RALTER  APPCLU             (TSTNET.TSTLOCLU.TSTPRTLU)  SESSION  (   LOCK        )"
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.extract_appc_session()`
+
+```python
+  def extract_appc_session(
+        self, net_id: str, local_lu: str, partner_lu: str
+  ) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+&nbsp;
+
+{: .experimental }
+> _So far, no tested **Segments** and **Traits** are considered **Stable** for profile extract of the **session** segment. This is how the function will work when **Traits** are made **Stable** for extract._
+
+&nbsp;
+
+Extract an existing **APPC Session** profile in the **APPCLU** class.
+
+#### 📥 Parameters
+* `class_name`<br>
+  The name of the **APPC Session** profile being defined to the **APPCLU** class.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Trait dictionary** of the values of the traits extracted from the **SESSION** segment of the Resource Profile or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+

```

Comment:
No example?

Reply:
This is deliberate as this function is so experimental there are currently no stable uses for it. This is specified in the Experimental tag for the Extract function.